packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.controller.js

```diff
@@ -290,4 +292,29 @@ export default class DedicatedServerDashboard {
   onBillingInformationError(error) {
     return this.Alerter.error(error, 'server_dashboard_alert');
   }
+
+  openOsInstallation(type) {
+    if (type === 'progress') {
+      this.trackClick(
+        'dedicated::dedicated::server::system-installation-progress',
```

Comment:
i think we are using a trackingPrefix variable in dedicated in order to simplify the management of tracking
it could be great if you could reuse or define a tracking prefix (if we need to move the section only the prefix will be updated)

Reply:
Not defined for `server` but there is an example of `trackingPrefix`here : 
https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/managedBaremetal/managed-baremetal.routing.js#L248
&